src/System.IO.FileSystem/src/System/IO/FromApp.Interop.cs

```diff
@@ -0,0 +1,150 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I know you commented on this in the PR description, but I'd still prefer to see these interop files live in src\Common\Interop.